clai/README.md

```diff
@@ -51,31 +51,63 @@ Either way, running `clai` will start an interactive session where you can chat
 - `/multiline`: Toggle multiline input mode (use Ctrl+D to submit)
 - `/cp`: Copy the last response to clipboard
 
-## Help
+## Web Chat UI
+
+Launch a web-based chat interface:
+
+```bash
+clai web -m openai:gpt-5
+```
+
+![Web Chat UI](https://github.com/user-attachments/assets/462d4647-a74d-4c35-9120-af0d682b04cf)
+
+This will start a web server (default: http://127.0.0.1:7932) with a chat interface.
 
+You can also serve an existing agent. For example, if you have an agent defined in `my_agent.py`:
+
+```python
+from pydantic_ai import Agent
+
+my_agent = Agent('openai:gpt-5', instructions='You are a helpful assistant.')
 ```
-usage: clai [-h] [-m [MODEL]] [-a AGENT] [-l] [-t [CODE_THEME]] [--no-stream] [--version] [prompt]
```

Comment:
Now that this section is gone, I think the main CLI docs need a table of available non-web CLI options, similar to the table we have in the web section